lib/Ray/include/Drawables/Basic/Circle.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Pixel
+*/
+
+#ifndef CIRCLE_HPP_
+#define CIRCLE_HPP_
+
+#include <raylib.h>
+#include "Drawables/Drawable.hpp"
+
+namespace Ray {
+    class Circle: public Drawable
+	{
+        public:
+            Circle(Vector2 topLeftPos, int radius, Color);
+            Circle(int topLeftX, int topLeftY, int radius, Color);
+            Circle(const Circle &);
+
+            Circle &operator=(const Circle &);
+
+            ~Circle() = default;
+
+            int getRadius(void) const;
+
+            bool collide(const Rectangle &);
```

Comment:
Collisions should not be handled by a Drawable.
Collisions should be handled by a PhysicSystem with it's own colliders class.

Reply:
Collision system between shapes was managed by raylib.
In this context, it is indeed not congruous